README.md

```diff
@@ -1 +1,148 @@
-# javascript-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Todo List Project](#-Javascript-project)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo video ](https://youtu.be/sgqM34DgA4Q)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Javascript Capstone<a name="about-project"></a>
+
+**javascript Capstone** 
+The JavaScript capstone project  is about building your own web application based on an external API. We select an API (The pokemon api) that provides data about our selected api and we build  the webapp around it. The webapp have 3 user interfaces.
+
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript(ES6)
+    Webpack
+    AAA patern
+    jest Tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo video](https://youtu.be/sgqM34DgA4Q)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Create a local directory that you want to clone the repository.
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command git clone ``
```

Comment:
- [ ] Good job for creating a counter for the comment and reservation but it would be better if each one have a separate module so it became easer to debug the code.
![image](https://user-images.githubusercontent.com/74744787/211160809-6859a24c-5d1c-4405-81fb-e48d87c74baa.png)